src/main/java/edu/kis/powp/jobs2d/drivers/adapter/Job2dAdapter.java

```diff
@@ -26,12 +27,14 @@ public void operateTo(int x, int y) {
 		ILine line = LineFactory.getBasicLine();
 		line.setStartCoordinates(this.startX, this.startY);
 		line.setEndCoordinates(x, y);
+		this.startX = x;
+        this.startY = y;
 
-		drawLine(line);
+		DrawerFeature.getDrawerController().drawLine(line);
```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.